dependencies.gradle

```diff
@@ -1,50 +1,50 @@
 ext {
     versions = [
             androidSdk     : [
-                    compile   : 28,
-                    min       : 21,
-                    target    : 28,
+                    compile: 28,
+                    min    : 21,
+                    target : 28,
             ],
             androidSupport : '28.0.0',
             butterknife    : '8.0.1',
             dexmaker       : '1.2',
             espressoSupport: '0.2.1',
-            room           : '1.0.0',
+            room           : '2.1.0-alpha02',
```

Comment:
I didn't quite like that it updated to the latest (or latest-ish) versions. This could cause some headaches but I guess there aren't older androidx versions.

While the migration guide says to update to support library 28, I would recommend upgrading **all the libraries that will be jetpack'd to the latest version** before `jetifying`.